modules/hcp-migrate-aws-multiarch-nodepools.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-migrate-aws-multiarch-nodepools_{context}"]
+= Creating node pools on the multi-architecture hosted cluster
+
+After transitioning your hosted cluster from single-architecture to multi-architecture, create node pools on compute machines based on 64-bit AMD and 64-bit ARM architectures.
+
+.Procedure
+
+. Create node pools based on 64-bit ARM architecture by entering the following command:
```

Comment:
Customers must complete steps 1 and 2 or do they have the option to create node pool son 1 arch type?